Jenkinsfile

```
@@ -357,7 +357,16 @@ pipeline {
                 }
             }
         }
+        stage('Benchmarks'){
+            steps{
+                withEnv(["PATH+=${GO}:${GOPATH}/bin"]){
+                    sh 'mkdir -p reports'
+                    sh "go test -timeout=20m -count=1 -run='^\044' -bench=Benchmark -test.benchmem -v ${SGW_REPO}/... > reports/benchmark.out || true"
```

The "standard" 'skip tests' flag I've seen is `-`
The "standard" 'all' flag I've seen is `.`
Don't need `test` prefix on benchmem flag

```suggestion
                    sh "go test -timeout=20m -count=1 -run=- -bench='.' -benchmem -v ${SGW_REPO}/... > reports/benchmark.out || true"
```